src/arc.ts

```diff
@@ -134,16 +134,25 @@ export async function initializeArc(provider?: any): Promise<boolean> {
 
     // https://www.apollographql.com/docs/link/links/retry/
     const retryLink = new RetryLink({
-      attempts: (count) => {
-        return (count !== 10);
+      attempts: {
+        max: 5,
+        retryIf: (error, _operation) => {
+        // eslint-disable-next-line no-console
+          console.error("error occurred fetching data, retrying...");
+          // eslint-disable-next-line no-console
+          console.log(error);
+          return !!error;
+        },
       },
-      delay: () => {
-        // This will give a random delay between retries between the range of 5 to 30 seconds.
-        return Math.floor(Math.random() * (30000 - 5000 + 1) + 5000);
+      delay: {
+        initial: 500, // this is the initial time after the first retry
+        // next retries )up to max) will be exponential (i..e after 2*iniitial, etc)
+        jitter: true,
+        max: Infinity,
       },
     });
 
-    arcSettings.graphqlRetryLink = retryLink;
+    arcSettings.retryLink = retryLink;
```

Comment:
this should be `arcSettings.graphqlRetryLink `. or do not set retryLink at all as arc.js expect `graphqlRetryLink`